validator/src/cli.rs

```diff
@@ -291,6 +291,19 @@ pub fn app<'a>(version: &'a str, default_args: &'a DefaultArgs) -> App<'a, 'a> {
                 .takes_value(true)
                 .help("Use DIR as snapshot location [default: --ledger value]"),
         )
+        .arg(
+            Arg::with_name("boot_from_local_state")
```

Comment:
I think it's fine because all the other args here do it. But it's always rubbed me a bit wrong that we've got nearly duplicate args here and in ledger-tool.

Only difference here is there's no conflict w/ "no_snapshot", since that's not a validator option.

@steviez (master of ledger-tool) are there any plans for consolidating between validator & ledger-tool?

Reply:
> are there any plans for consolidating between validator & ledger-tool?

Nothing concrete, but I share your sentiment about the duplication and have thought about this in the past. There is a little gray area given that validator and ledger-tool are very similar, but also slightly different here and there (like your comment "Only difference here is there's no conflict w/ "no_snapshot", since that's not a validator option.")

I have a branch that I haven't touched in a bit that worked on cleaning up the ledger-tool args in a way similar to validator. That was step 1 in my mind, and then finding a way/place (maybe sdk) to share between ledger-tool and validator was step 2.

Maybe I'll brush the dust off the step 1 PR to get the ball rolling on this. Code deduplication aside, it has annoyed me for quite some time that `cargo fmt` breaks down in `ledger-tool`.